From f9c2ae13a80f3b0f309214c6b57bc3a705b71398 Mon Sep 17 00:00:00 2001
From: Ryan Eatmon <reatmon@ti.com>
Date: Mon, 3 Feb 2025 16:54:24 -0600
Subject: [tiU25.01]] remoteproc: k3-m4: Fix implicit definition of
 ti_secure_image_post_process

Add missing include to make sure that ti_secure_image_post_process() is
not implicitly declared.

Signed-off-by: Ryan Eatmon <reatmon@ti.com>

Upstream-Status: Pending

---
 drivers/remoteproc/ti_k3_m4_rproc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/remoteproc/ti_k3_m4_rproc.c b/drivers/remoteproc/ti_k3_m4_rproc.c
index 5a5ae2fa81d..516c817385e 100644
--- a/drivers/remoteproc/ti_k3_m4_rproc.c
+++ b/drivers/remoteproc/ti_k3_m4_rproc.c
@@ -20,6 +20,7 @@
 #include <linux/sizes.h>
 #include <linux/soc/ti/ti_sci_protocol.h>
 #include "ti_sci_proc.h"
+#include <mach/security.h>
 
 /**
  * struct k3_m4_mem - internal memory structure
-- 
2.17.1

